src/js/button.js

```diff
@@ -70,7 +70,7 @@ class Button extends Component {
   // KeyPress (document level) - Trigger click when keys are pressed
   onKeyPress(event) {
     // Check for space bar (32) or enter (13) keys
-    if (event.which == 32 || event.which == 13) {
+    if (event.which === 32 || event.which === 13) {
```

Comment:
Yea sorry about that. I should have left the eqeqeq option in jslint for another pull request. I'm trying to fix the unit tests ASAP so we could merge quickly.